ethcore/src/pod_account.rs

```diff
@@ -96,7 +96,7 @@ impl From<ethjson::spec::Account> for PodAccount {
 		PodAccount {
 			balance: a.balance.map_or_else(U256::zero, Into::into),
 			nonce: a.nonce.map_or_else(U256::zero, Into::into),
-			code: a.code.map(Into::into).or(Some(Vec::new())),
+			code: a.code.map(Into::into).or_else(|| Some(Vec::new())),
```

Comment:
`or_else` is the lazy version of `or`. Creating an empty vector on stack costs almost nothing though.